opaque_keys/edx/keys.py

```diff
@@ -77,6 +77,26 @@ def block_type(self):  # pragma: no cover
         raise NotImplementedError()
 
 
+class AggregateCourseKey(OpaqueKey):
+    """
+    An :class:`opaque_keys.OpaqueKey` identifying a
+    serialized Course Key object.
+    """
+    KEY_TYPE = 'aggregate_course_key'
```

Comment:
Is this something we should do as part of this PR?  We can certainly make the change but I don't want to introduce two new ideas at one time if we don't have to